### PR TITLE
fix: publish nightly action update

### LINKED ISSUE #420: bug: actions not working (builds) (publish)

### PR DESCRIPTION
**Pull request by esavier**
fixes #420

### CODE REVIEW

File: .github/workflows/publish-nightly.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ jobs:
 
     - name: Build and push crates
       run: |
-        crates=(command-service data-router leader-elector query-service query-service-ts schema-registry query-router api)
+        crates=(command-service data-router query-service query-service-ts schema-registry query-router api edge-registry partial-update-engine object-builder materializer)
```
Comment (frondeus) [inline .github/workflows/publish-nightly.yml:48]: shouldnt it be materializer-general and ondemand?

Comment (esavier) [reply .github/workflows/publish-nightly.yml:48]: this is from devel xD

**Review by luke-biel — DISMISSED** [review]

File: .github/workflows/publish-nightly.yml (hunk 1 above)
Comment (esavier) [reply .github/workflows/publish-nightly.yml]: done

File: .github/workflows/publish-nightly.yml [hunk 2]
```diff
@@ -45,7 +45,7 @@ jobs:
 
     - name: Build and push crates
       run: |
-        crates=(command-service data-router leader-elector query-service query-service-ts schema-registry query-router api)
+        crates=(data-router command-service query-router query-service query-service-ts schema-registry upload-to-kafka upload-to-rabbitmq api edge-registry partial-update-engine object-builder materializer-general materializer-ondemand)
```
Comment (luke-biel) [inline .github/workflows/publish-nightly.yml:48]: We can't push `upload-*` to hub - there's  no such repos. Also, I'm not sure if we wanna to.

**Review by luke-biel — CHANGES_REQUESTED** [review]

File: .github/workflows/publish-nightly.yml (hunk 2 above)
Comment (esavier) [reply .github/workflows/publish-nightly.yml:48]: right

Comment (esavier) [reply .github/workflows/publish-nightly.yml]: done

**Review by frondeus — APPROVED** [review]

**Pull request merged**